project/src/main/world/OverworldIndoors.tscn

```diff
@@ -690,7 +696,6 @@ dna = {
 "shader:TailZ1:red": Color( 0.560784, 0.105882, 0.129412, 1 ),
 "tail": "5"
 }
-elevation = 150
```

Comment:
Why was this flag removed? Is the creature still drawn correctly?

Reply:
Restored `elevation-150` value; the creature was being drawn on the floor instead of on the stool.